plenum/server/domain_req_handler.py

```diff
@@ -38,16 +38,16 @@ def validate(self, req: Request, config=None):
                                                 req.reqId,
                                                 error)
 
-    def _reqToTxn(self, req: Request):
-        txn = reqToTxn(req)
+    def _reqToTxn(self, req: Request, tm: int):
+        txn = reqToTxn(req, tm)
         for processor in self.reqProcessors:
             res = processor.process(req)
             txn.update(res)
 
         return txn
 
-    def apply(self, req: Request):
-        txn = self._reqToTxn(req)
+    def apply(self, req: Request, tm: int):
```

Comment:
"tm" is the name of a time struct in python and C. Seeing this variable name, "tm" causes me to think one thing, and then its datatype surprises me. It might be better to name this parameter "secs_since_epoch" or something more obvious.